### PR TITLE
Move storage/auth/auditLogger/theme into launcher

### PR DESCRIPTION
**Pull request by knolleary**
Part of #99 

## Description

This moves the following repos into this one:
 
- `@flowforge/nr-audit-logger`
 - `@flowforge/nr-auth`
 - `@flowforge/nr-storage`
 - `@flowforge/nr-theme`

 I have test this locally, but experience tells me the paths around module loading at subtly different in docker/k8s land. Getting this raised now to get some eyes on it, but will want to finalise some docker-based testing before we merge.

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [-] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Review by hardillb — COMMENTED** [review]
Should the icons be included twice? Once in `theme/common` and again in `resouces`

**Comment by Steve-Mcl** [comment]
> > Should the icons be included twice?
> 
> Possibly not. They are in both places in the original repo, but I can't immediately see any use of them from the common folder.
> 
> May need @Steve-Mcl to confirm, but I think we can remove the images from `lib/theme/common` as they only ever get loaded from `resources/`

I have a vague memory of changing things around to use the resources but may not have "clean up" afterwards.

The idea was (at the time) that the theme can have own icon or fall back to common but I am unsure where it got to in the end. (sorry)

**Comment by hardillb** [comment]
OK, I'm going to test this in a container this afternoon, assuming no other issues I'll test with/without the extra images and then merge as appropriate

File: package.json [hunk 1]
```diff
@@ -27,9 +43,6 @@
     },
     "homepage": "https://github.com/flowforge/flowforge-nr-launcher#readme",
     "dependencies": {
-        "@flowforge/nr-audit-logger": "^1.3.0",
-        "@flowforge/nr-auth": "^1.3.0",
-        "@flowforge/nr-storage": "^1.3.0",
         "@flowforge/nr-theme": "^0.1.3",
```
Comment (hardillb) [inline package.json:46]:
```suggestion
```

I assume this should also have been removed

**Comment by hardillb** [comment]
OK, so I removed the `nr-theme` line mention in the `package.json` dependencies and tested.

Auth, Storage and Audit logging appear to be working, but the theme has not been loaded.

**Comment by hardillb** [comment]
Team libraries are also not working

**Comment by hardillb** [comment]
I think I've fixed it

**Comment by knolleary** [comment]
Okay - so I think the only remaining task is to remove the duplicate icons. I've test that locally and if it looks good, will push then get this merged.

**Pull request merged**